msu/hooks/entity/world/settlement.nut

```diff
@@ -0,0 +1,18 @@
+::mods_hookExactClass("entity/world/settlement", function(o) {
+	local onEnter = o.onEnter;
+	o.onEnter = function()
+	{
+		local ret = onEnter();
+
+		if (ret)
+		{
+			local roster = this.World.getPlayerRoster().getAll();
+			foreach (bro in roster)
+			{
+				bro.getSkills().onEnterSettlement(this);
+			}
+		}
+
+		return ret;
+	}
+});
```

Comment:
someone doesn't have editorconfig set up

Reply:
Hmm... I wonder who that must be :-s